### PR TITLE
HBASE-21433 Add Apache Yetus integration for hbase-operator-tools repository

### PR DESCRIPTION
**Pull request by petersomogyi**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/PreCommit-HBASE-OPERATOR-TOOLS-Build/1/

**Comment by petersomogyi** [comment]
Trying out 2 commits in a single PR.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/PreCommit-HBASE-OPERATOR-TOOLS-Build/2/

**Comment by petersomogyi** [comment]
Master branch does not define ${maven.javadoc.version} and ${maven.javadoc.version} that's why javadoc steps failed on master but passed on this PR.

File: pom.xml [hunk 1]
```diff
@@ -254,31 +252,19 @@
               <exclude>**/.idea/**</exclude>
               <exclude>**/*.iml</exclude>
               <exclude>**/CHANGES.txt</exclude>
+              <exclude>README.md</exclude>
```
Comment (busbey) [inline pom.xml]: could we fix the header instead?

Comment (petersomogyi) [reply pom.xml]: Added license to README files.

**Pull request merged**